tests/lean/ppMotives.lean.expected.out

```diff
@@ -22,7 +22,7 @@ fun x x_1 x_2 x_3 =>
   | α, .(α), Eq.refl α, a => HEq.refl _
 theorem ex.{u} : ∀ {α β : Sort u} (h : α = β) (a : α), cast h a ≅ a :=
 fun x x_1 x_2 x_3 =>
-  match x, x_1, x_2, x_3 : Sort u → Sort u → ∀ x_5, ∀ x_6, cast x_5 x_6 ≅ x_6 with 
+  match x, x_1, x_2, x_3 : ∀ (x x_4 : Sort u) (x_5 : x = x_4) (x_6 : x), cast x_5 x_6 ≅ x_6 with 
```

Comment:
Currently, `pp.analyze` doesn't actually see the `pi` type that gets annotated for matches, it only sees its original `lambda` version. So, it does not annotate binder types. The new behavior is better -- the "synthetic" `pi` now gets the binder types despite `pp.analyze` not annotating them.